.profile.d/heroku-setup.sh

```diff
@@ -28,7 +28,7 @@ echo "ServerTokens Prod" >> /app/vendor/heroku/heroku-buildpack-php/conf/apache2
 
 # PHP configuration
 # Modifications to conf/php/php.ini from the Heroku PHP buildpack
-sed -i 's/^expose_php = On/expose_php = Off/' /app/vendor/heroku/heroku-buildpack-php/conf/php/php.ini
+# sed -i 's/^expose_php = On/expose_php = Off/' /app/vendor/heroku/heroku-buildpack-php/conf/php/php.ini
```

Comment:
This shouldn't be necessary anymore. This option (to turn off `expose_php`) was added to the default buildpack in https://github.com/heroku/heroku-buildpack-php/commit/d905955e7e0cbc1011e1f1c42a207d234b4ac694